### ISSUE TITLE
Occasionally won't show reddit posts in comments

### ISSUE THREAD

**Issue by LZetterlund** [xlexi/alientube#160]
One example is this video, https://www.youtube.com/watch?v=lL-WMetAa7c I know this has been submitted to reddit here, https://www.reddit.com/r/trapmuzik/comments/3fv5wt/underrated_skeme_36_oz/ but is not showing up in the comments section. 

Thanks for making a great extension!

**Comment by HandsOfBlue** [comment]
I, too, am having this problem, as is a friend of mine. The YouTube comments will show up, and refreshing the page will load the Reddit comments. AlienTube was working fine for me until yesterday; for my friend, it has not worked properly since he installed it.

My details: Version 46.0.2490.86 m (64-bit)

**Comment by hockeygoalie5** [comment]
Which browser are you using? The comments on that video loaded properly for /r/hiphopheads and /r/trapmuzik for me.

**Comment by HandsOfBlue** [comment]
...whoops, that IS pertinent info, isn't it?

I'm on Chrome.

**Comment by hockeygoalie5** [comment]
The linked video worked on that same version of Chrome as well. It consistently does not load comments until you refresh?

**Comment by HandsOfBlue** [comment]
Right; I feel like as it's a problem which just sort of "popped up" one
day, apropos nothing, it'll be hard to reproduce (barring any hunches)...

Sorry I couldn't be more helpful.

**Comment by hhu94** [comment]
I'm experiencing the same problem, it seems like the extension just doesn't load sometimes. However when I refresh it always works.
